rte_mis.install

```diff
@@ -40,3 +40,47 @@ function rte_mis_install() {
   }
 
 }
+
+/**
+ * Impelments hook_schema().
+ */
+function rte_mis_schema() {
+  $schema['lottery_school_seats_status'] = [
+    'description' => 'Stores the school seat status during the lottery',
+    'fields' => [
+      'school_name' => [
+        'description' => 'School Name',
+        'type' => 'varchar',
+        'length' => 255,
+        'not null' => TRUE,
+      ],
+      'school_id' => [
+        'description' => 'School Id',
+        'type' => 'int',
+        'unsigned' => TRUE,
```

Comment:
@RohanSingh-inno Why have we created table `lottery_school_seats_status`  at the profile level?